### DISCUSSION TITLE
Adding the imported issue key to the storySchema

### DISCUSSION THREAD

**Discussion by cdwillie76** [Zuehlke/poinz#320] [Ideas]
I plan on creating a script to take the exported JSON file as input to update the point values on Jira issues on our internal Jira instance.  Right now it looks like I could parse the description to pull out the key from the URL.  Would the project be open to an update to include the imported issue key in the storySchema so it could be exported in the JSON?

I am willing to work on a pull request if this is a feature that would be considered to be added to poinz.

Thanks!
Chris

**Comment by xeronimus** [comment]
Thanks for the suggestion :)

some thoughts and questions:

- currently the issue-key is concatenated with the title (during import), and not stored separately... (see  
https://github.com/cdwillie76/poinz/blob/b3fc4147eba1c20a28587d5e7ba8cd1cc8025d61/server/src/commandHandlers/parseCsvDataUrlToStories.js)
- if we extend the schema with a "key" ("story-key"  / "issue-key" ), this should be a optional field (not mandatory)
- Would you also provide the user with an edit-field (like title and description) in the story-edit form?  Or would this "key" field only be a "hidden" field, that is only used when importing from csv-files?

**Reply by cdwillie76** [reply]
> currently the issue-key is concatenated with the title (during import), and not stored separately... (see
https://github.com/cdwillie76/poinz/blob/b3fc4147eba1c20a28587d5e7ba8cd1cc8025d61/server/src/commandHandlers/parseCsvDataUrlToStories.js)

Thanks.  I saw it was used for the URL if present and missed it was added to the title

>if we extend the schema with a "key" ("story-key" / "issue-key" ), this should be a optional field (not mandatory)

Concur

>Would you also provide the user with an edit-field (like title and description) in the story-edit form? Or would this "key" field only be a "hidden" field, that is only used when importing from csv-files?

I was thinking hidden and only used during the import of csv-files.

Thanks and Happy New Year!

**Reply by xeronimus** [reply]
you can check it out on #323
